lib/active_record/connection_adapters/oracle_enhanced/structure_dump.rb

```diff
@@ -175,8 +175,9 @@ def structure_dump_table_comments(table_name)
         def structure_dump_column_comments(table_name)
           comments = []
           columns = select_values(<<~SQL.squish, "SCHEMA")
-            SELECT /*+ OPTIMIZER_FEATURES_ENABLE('11.2.0.2') */ column_name FROM user_tab_columns
-            WHERE table_name = '#{table_name}' ORDER BY column_id
+            SELECT /*+ OPTIMIZER_FEATURES_ENABLE('11.2.0.2') */ column_name FROM all_tab_cols
```

Comment:
Would you explain why `all_tab_cols` is used not `all_tab_columns`?

Reply:
That's my ignorance, sorry! Having read [some docs](https://docs.oracle.com/en/database/oracle/oracle-database/12.2/refrn/ALL_TAB_COLS.html#GUID-85036F42-140A-406B-BE11-0AC49A00DBA3) I will swap this out :)

Reply:
I've force-pushed with this swapped out, thanks for the feedback. 👍